content/blog/2017/2017-01-12-declarative-pipeline-beta-2.adoc

```diff
@@ -0,0 +1,151 @@
+[pipeline]
+----
+// Declarative //
+agent {
+    docker {
+        image "ubuntu:16.04"
+        label "docker-label"
+        args "-v /tmp:/tmp -p 8000:8000"
+    }
+}
+----
+
+When you're building an image from "Dockerfile" in your repository and
+don't care what node is used or have additional arguments, you can again
+use a simple syntax.
+[pipeline]
+----
+// Declarative //
+agent {
+    dockerfile true
+}
+----
+
+When you're building an image from a different file, or have a label or other
+arguments, use the following syntax:
+[pipeline]
+----
+// Declarative //
+agent {
+    dockerfile {
+        filename "OtherDockerfile"
+        label "docker-label"
+        args "-v /tmp:/tmp -p 8000:8000"
+    }
+}
+----
+
+=== `when` - conditional stage execution
+We introduced the `when` section a couple releases ago, but have made some
+changes to its syntax here in 0.8.1. We wanted to add some simpler ways to
+specify common conditions, and that required we re-work the syntax accordingly.
+Here's what `when` looks like now:
+
+One of the most common conditions is running a stage only if you're on a
+specific branch. You can also use wildcards like "*/master".
+[pipeline]
+----
+// Declarative //
+when {
+    branch "master"
+}
+----
+
+Another built-in condition is the environment condition, which checks to see
+if a given environment variable is set to a given value.
+[pipeline]
+----
+// Declarative //
+when {
+    environment name: "SOME_ENV_VAR", value: "SOME_VALUE"
+}
+----
+
+Lastly, there's the expression condition, which resolves an arbitrary
+Pipeline expression. If the return value of that expression isn't false or
+null, the stage will execute.
+[pipeline]
+----
+// Declarative //
+when {
+    expression {
+        echo "Should I run?"
+        return "foo" == "bar"
+    }
+}
+----
+
+=== Properties, options and wrappers
+We've renamed the old `properties` section to `options`. The `options` section
+is now where you'll put general Pipeline options like `buildDiscarder`,
+Declarative-specific options like `skipDefaultCheckout`, and block-scoped steps
+that should wrap around the execution of the entire build, like `timeout` or
+`timestamps`.
+
+[pipeline]
+----
+// Declarative //
+
+options {
+    buildDiscarder(logRotator(numToKeepStr:'1'))
+    skipDefaultCheckout()
+    timeout(time: 5, unit: 'MINUTES')
+}
+----
+
+== Heading towards 1.0!
+While we may still add more functionality to the Declarative Pipeline syntax
+before the 1.0 release, we won't be changing any existing syntax until, at the
+earliest, a future 2.0 release. This means that any pipelines you write against
+the 0.8.1 syntax will keep working for the foreseeable future without any
+changes. So if you're already using Declarative Pipelines, make sure to update
+your `Jenkinsfile`s after upgrading to 0.8.1, and if you haven't been using
+Declarative Pipelines yet, install the _Pipeline: Model Definition_ plugin and
```

Comment:
Make this a link too.  I know you have one earlier, but this is the lead out - you're telling them where to got next, give them a link.